doc/build/dts/bindings-upstream.rst

```diff
@@ -38,49 +41,59 @@ In particular, this rule applies if:
+- If the binding is bus-specific, you can append the bus to the file name;
+  for example, if the binding YAML has ``on-bus: bar``, you may name the file
+  ``vnd,foo-bar.yaml``.
 
-  - For example, a binding for compatible ``vnd,foo`` must be named ``vnd,foo.yaml``.
-  - If the binding is bus-specific, you can append the bus to the file name;
-    for example, if the binding YAML has ``on-bus: bar``, you may name the file
-    ``vnd,foo-bar.yaml``.
+Recommendations are requirements
+================================
 
-- All recommendations in :ref:`dt-bindings-default` are requirements when
-  submitting the binding.
+All recommendations in :ref:`dt-bindings-default` are requirements when
+submitting the binding.
 
-  In particular, if you use the ``default:`` feature, you must justify the
-  value in the property's description.
+In particular, if you use the ``default:`` feature, you must justify the
+value in the property's description.
 
-- There are two ways to write property ``description:`` strings that are always
-  OK.
+Descriptions
+============
 
-  If your description is short, it's fine to use this style:
+There are only two acceptable ways to write property ``description:``
+strings.
 
-  .. code-block:: yaml
+If your description is short, it's fine to use this style:
 
-     description: my short string
+.. code-block:: yaml
+
+   description: my short string
 
-  If your description is long or spans multiple lines, you must use this
-  style:
+If your description is long or spans multiple lines, you must use this
+style:
 
-  .. code-block:: yaml
+.. code-block:: yaml
 
-     description: |
-       My very long string
-       goes here.
-       Look at all these lines!
+   description: |
+     My very long string
+     goes here.
+     Look at all these lines!
 
-  This ``|`` style prevents YAML parsers from removing the newlines in
-  multi-line descriptions. This in turn makes these long strings
-  display properly in the :ref:`devicetree_binding_index`.
+This ``|`` style prevents YAML parsers from removing the newlines in
+multi-line descriptions. This in turn makes these long strings
+display properly in the :ref:`devicetree_binding_index`.
 
-  Do not use any other style for long or multi-line strings.
+Naming conventions
+==================
 
-- Do not use uppercase letters (``A`` through ``Z``) or underscores (``_``) in
-  property names. Use lowercase letters (``a`` through ``z``) instead of
-  uppercase. Use dashes (``-``) instead of underscores. (The one exception to
-  this rule is if you are replicating a well-established binding from somewhere
-  like Linux.)
+Do not use uppercase letters (``A`` through ``Z``) or underscores (``_``) in
+property names. Use lowercase letters (``a`` through ``z``) instead of
+uppercase. Use dashes (``-``) instead of underscores. (The one exception to
+this rule is if you are replicating a well-established binding from somewhere
+like Linux.)
```

Comment:
Should we recommend using all-lowercase for enum property values as well?

Reply:
Is there a linux convention for that?

Reply:
I believe so, but I am have not been able to find any documentation for it.

Reply:
I think I'd rather leave it out for now, then, since it's ambiguous.